topydo/ui/columns/CompletionBoxWidget.py

```diff
@@ -0,0 +1,58 @@
+# Topydo - A todo.txt client written in Python.
+# Copyright (C) 2015 Bram Schoenmakers <bram@topydo.org>
```

Comment:
Copyright is out of date. You may add your own name as well, as its not my code.